spring-cloud-gcp-autoconfigure/src/main/java/org/springframework/cloud/gcp/autoconfigure/datastore/EmulatorSettings.java

```diff
@@ -0,0 +1,65 @@
+/*
+ * Copyright 2017-2019 the original author or authors.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *      https://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package org.springframework.cloud.gcp.autoconfigure.datastore;
+
+/**
+ * Properties for configuring Cloud Datastore Emulator.
+ *
+ * @author Lucas Soares
+ *
+ * @since 1.2
+ */
+public class EmulatorSettings {
+	/**
+	 * Is datastore emulator auto configuration enabled. If enabled the Datastore client
+	 * instance itself will point to the local Datastore instance with the specified
+	 * {@link #port}.
+	 */
+	private boolean enabled;
+
+	/** Is the datastore emulator port. Default: 8081 */
+	private int port = 8081;
+
+	/**
+	 * Consistency to use creating the Datastore server instance. Default: <code>0.9</code>
```

Comment:
It is not recommended in the modern JavaDocs to to use HTML tags at all. The `{@code}` must be used instead.
More over no any JavaDoc hints in this properties descriptions.